spec/linter-registry-spec.coffee

```diff
@@ -0,0 +1,169 @@
+describe 'linter-registry', ->
+  LinterRegistry = require('../lib/linter-registry')
+  EditorLinter = require('../lib/editor-linter')
+  beforeEach ->
+    waitsForPromise ->
+      atom.workspace.destroyActivePaneItem()
+      atom.workspace.open('test.txt')
+  describe '::construct', ->
+    it 'expects no arguments', ->
+      linterRegistry = new LinterRegistry
+      expect(true).toBe(true)
+      linterRegistry.deactivate()
+  describe '::addLinter', ->
```

Comment:
Whitespace is your friend and makes everything more readable.  Add some between describe blocks and test blocks